pyfiglet/__init__.py

```diff
@@ -819,7 +860,13 @@ def main():
         # Set stdout to binary mode
         sys.stdout = sys.stdout.detach()
 
-    sys.stdout.write((r + '\n').encode('UTF-8'))
+    if ':' not in opts.color:
+        opts.color += ':'
+    foreground_color = __color_to_ansi(opts.color.split(':', 1)[0], isBackground=False)
+    background_color = __color_to_ansi(opts.color.split(':', 1)[1], isBackground=True)
+    colors_string = foreground_color + background_color
+    reset_colors = '' if colors_string == '' else  __color_to_ansi('RESET', False)
+    sys.stdout.write((colors_string + (r + '\n') + reset_colors).encode('UTF-8'))
```

Comment:
Here is how I would prefer this looked (below). Notice that, even on a skim read,  it's clear this example code what the effect is if the color is unspecified. There are repeated calls to `sys.stdout.write`, which might seem verbose or wasteful but makes it obvious to the reader what is happening. I think it is harder to do so in the code as it is currently written, in part because there is a lot happening within each line.

Note that `pring_figlet` should be adjusted likewise.

```python
colors = parse_colors(opts.color)

if colors is not None:
    sys.stdout.write(colors)

sys.stdout.write(r.encode('UTF-8'))
sys.stdout.write('\n')

if colors is not None:
    sys.stdout.write(reset_colors)
```

Reply:
More readable is always better!